manifests/image-references

```diff
@@ -42,6 +42,11 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-ovn-kubernetes:latest
+# NOTE: Skipped until the image is added to an actual release
```

Comment:
I think this is what's causing a CI failure, you probably need this.

But, now it's going to probably complain that it's an image that's external to the release.

Reply:
It was complaining about not being able to find the image nevertheless before that. From my conversations with @squeed it does seem that that's an alias that searches for the actual url, once the image is on a release, so probably stuck on that dependency.

Reply:
``` info: Included 122 images from 35 input operators into the release
error: unable to create a release: operator "cluster-network-operator" contained an invalid image-references file: no input image tag named "egress-router-cni" ``` This was the issue with that block before putting that out.